db/migrate/20150328224118_add_api_key_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddApiKeyToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :api_key, :string
+  end
+end
```

Comment:
Recommed adding another migration to put a unique constraint on the api_key column.